src/rprblender/nodes/blender_nodes.py

```diff
@@ -1051,32 +1051,24 @@ def export(self) -> [NodeItem, None]:
         roughness_radial = self.get_input_value('Radial Roughness').clamp(0.001, 1.0)
         coat = self.get_input_scalar('Coat')
         ior = self.get_input_scalar('IOR')
+        weight = self.node_item(0.0)
+        melanin = self.node_item(0.0)
```

Comment:
it can be simplified by assigning float value, no need to use node_item()

Reply:
Fixed